### ISSUE TITLE
[BUG] Error when running action

### ISSUE THREAD

**Issue by svallory** [specfy/stack-analyser#87]
**Describe the bug**
Github complains of missing `token` input. There's no documentation saying that a `token` parameter is needed or where to get that token. Looking at the code, there's also a `GITHUB_WORKSPACE` env variable that I don't know how to setup 

**To Reproduce**
Copy the code for the action from the marketplace and paste it in the workflow

**Comment by bodinsamuel** [comment]
Hey, 
Thanks for raising the issue. The repo was initially reaching Specfy's backend to upload the results. To make it usable by other tool I made it agnostic but I during the refactor I forgot the GitHub actions token, sorry about that.

I'm fixing this right now. I'm curious what is your use case if you can share? ☺️

**Comment by svallory** [comment]
> I'm fixing this right now. I'm curious what is your use case if you can share?

Sorry for the delay. I was trying to use it to evaluate a project I was asked to complete.

> edit: should be good now, no need to specify any arguments

Are you sure? I added my token to make it work after reading the source code

**Comment by bodinsamuel** [comment]
> Are you sure? I added my token to make it work after reading the source code

Yes it's not required to execute the tech detection. Then if you want to use Specfy platform, you don't need anything, just go to specfy.io > connect your github org and import a repo directly from the UI
